_guide/semantic-html.md

```diff
@@ -11,7 +11,11 @@ roles:
 - Developer
 ---
 
+<<<<<<< HEAD
 Writing semantic HTML can make your site more accessible. When you switch your browser to "read mode", only the main content will be visible. Read mode hides:
+=======
+Writing semantic HTML can make your site more accessible. When you switch your browser to "read mode", only the main content will be visible. Read mode hides:
+>>>>>>> 11f717f (adding page to guide for semantic html)
```

Comment:
@jaube05 looks like we got some merge commit code added.I think it just needs manual clearing up.

Reply:
Fixed